src/vnc/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project(VncServer C)
 
 add_compile_options(-Wall)
 
+if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
```

Comment:
Quick comment: `MATCHES` seems more appropriate for regexp. Maybe use `EQUAL` instead?

Reply:
Yes agreed, replaced by STREQUAL